.github/workflows/datadog-synthetics.yml

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "Damarcus Jones Sr",
+  "img": "https://avatars.githubusercontent.com/u/57936?v=4",
+  "email": "jonestechnologyfoundation@gmail.com",
+  "links": {https://buymeacoffee.com/damarcusjoo
+    "website": "example.com",
+    "linkedin": "https://www.linkedin.com/in/example",
+    "github": "https://github.com/ijunxyz123"
+  },
+  "jobTitle": "Software Developer",
+  "location": {Cleveland Ohio 
+    "city": "Cleveland",
+    "state": "Ohio",
+    "country": "United States"
+  }
+}# This workflow will trigger Datadog Synthetic tests within your Datadog organisation
```

Comment:
Please remove all the code related to the workflow. Only add the json object in the correct file format instead of yaml.